catalog/pytest.ini

```diff
@@ -18,20 +18,13 @@ addopts =
     --disable-socket
     --allow-unix-socket
 
-# sqlalchemy
-#   Warning in dependency, nothing we can do
-#   https://docs.sqlalchemy.org/en/20/errors.html#error-b8d9
-# airflow
-#   Warning within Airflow for an internal call, nothing we can do
 # flask
 #   https://docs.sqlalchemy.org/en/20/errors.html#error-b8d9
 #   Warning in dependency, nothing we can do
 #   "removed"/"remoevd" is due to https://github.com/pallets/flask
 # distutils
 #   Warning in dependency, nothing we can do
 filterwarnings=
-    ignore::sqlalchemy.exc.MovedIn20Warning
```

Comment:
After rebasing locally and running `just catalog/test`, I see this MovedIn20Warning. Do we still need this filtered?

Reply:
Just to be sure, did you run a `just build` after rebasing?

Reply:
I did a full `recreate`, actually, and confirmed locally that I was on Airflow v2.8.0 :/

Reply:
Oh my goodness, I realized I had `SQLALCHEMY_SILENCE_UBER_WARNING=1` in my catalog environment file 🤦🏼‍♀️ It seems reasonable to silence that in general so I'll add that to this PR! Thanks for pointing that out! The GHA test outputs also show that warning FWIW.

Reply:
Confirmed, warnings should now no longer be shown! Though you'll probably need to add that variable to your env file first Staci 😄

Reply:
Oh, that'll do it! 😄 Looks great, makes sense to me.